plugins/woocommerce/client/admin/client/settings-payments/utils.ts

```diff
@@ -63,3 +63,15 @@ export const providersContainWooPaymentsInTestMode = (
 		!! wooPayments?.state?.test_mode && ! wooPayments?.state?.needs_setup
 	);
 };
+
+/**
+ * Checks whether providers contain WooPayments gateway in dev mode that is set up.
+ *
+ * @param providers payment providers
+ */
+export const providersContainWooPaymentsInDevMode = (
```

Comment:
This name is definitely too jargony. I suggest simply `isWooPaymentsInDevMode`.

Reply:
maybe in future PR! it's the same with the sandbox mode